CONTRIBUTING.md

```diff
@@ -30,3 +30,39 @@ It will also auto-generate user votes over time for the questions there.
 
 If you're curious about the technologies used in the server and client,
 see their respective `README.md` files.
+
+To run tests against a DynamoDB instance running [locally](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.html), make sure
+you got [`docker`](https://docs.docker.com/engine/install/) and
+[`AWS CLI`](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html#getting-started-install-instructions) installed, then hit:
+
+```console
+$ cd server
+$ ./run-dynamodb-local.sh
+```
+
+This will also spin a [Web UI](https://github.com/aaronshaf/dynamodb-admin?tab=readme-ov-file)
+for your local DynamoDB instance.
+
+You can now run tests with:
+
+```sh
+AWS_DEFAULT_REGION=us-east-1 \
+   AWS_ACCESS_KEY_ID=lorem \
+   AWS_SECRET_ACCESS_KEY=ipsum \
+   AWS_ENDPOINT_URL=http://localhost:8000 \
```

Comment:
Should we set it up so that there's just one environment variable to set (like `USE_DYNAMODB=test`), and when it's set like that then we set the others accordingly?

Reply:
I _think_ this setup is possible: we will need to check the `USE_DYNAMODB` variable and, if it is set to `test`, construct the [`config`](https://docs.rs/aws-config/latest/aws_config/struct.SdkConfig.html) accordingly using the [`builder`](https://docs.rs/aws-types/1.3.3/aws_types/sdk_config/struct.Builder.html). Let me look into into it.